### PR TITLE
WAI-946 Part 1: change mapOverlays table to snake_case

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #:
WAI-946

### CODE REVIEW

**Review by EMcQ-BES — APPROVED** [review]
What a very boring thing to review 😂

Change all look good, naturally I'm more concerned about the possibility of spots that haven't changed but should have. But looks like you've been very thorough 👍

**Comment by edmofro** [comment]
Closing in favour of https://github.com/beyondessential/tupaia/pull/3273

File: packages/web-config-server/src/apiV1/measureData.js [hunk 1]
```diff
@@ -334,17 +328,19 @@ export default class extends DataAggregatingRouteHandler {
...
 
     const entityCode = shouldFetchSiblings
       ? await this.getCountryLevelOrgUnitCode()
       : this.entity.code;
     const entity = await this.models.entity.findOne({ code: entityCode });
-    const dataServices = createDataServices(mapOverlay);
-    const dhisApi = getDhisApiInstance({ entityCode: this.entity.code, isDataRegional });
+    const dhisApi = getDhisApiInstance({
+      entityCode: this.entity.code,
+      isDataRegional: dataServices?.[0]?.isDataRegional,
```
Comment (EMcQ-BES) [inline packages/web-config-server/src/apiV1/measureData.js:342]: That optional chaining for arrays came up again soon!

**Review by EMcQ-BES — APPROVED** [review]

File: packages/web-config-server/src/apiV1/measureData.js (hunk 1 above)
Comment (edmofro) [reply packages/web-config-server/src/apiV1/measureData.js:342]: Yeah was a time saver having just gone through that review or I guarantee I would've got it wrong

**Pull request closed without merging**